code/game/atoms.dm

```diff
@@ -766,6 +766,9 @@ its easier to just keep the beam vertical.
 /atom/movable/proc/checkpass(passflag)
 	return pass_flags&passflag
 
+/atom/movable/proc/check_cantpass(passflag) //Currently only implemented for PASSMOB (mob_movement.dm)
```

Comment:
May implement it for the other flags if nobody suggests a better way to do this and if necessary